web/src/components/Plugin/PluginDetail.tsx

```diff
@@ -121,6 +122,12 @@ const PluginDetail: React.FC<Props> = ({
     if (name === 'cors') {
       const formData = UIForm.getFieldsValue();
       const newMethods = formData.allow_methods.join(",");
+      const compactAllowRegex = compact(formData.allow_origins_by_regex);
```

Comment:
Hi, we may add the test case for this case.

Reply:
Yes yes

Reply:
> Hi, we may add the test case for this case.

ok, on the way

Reply:
fixed